.ci/ci.yaml

```diff
@@ -0,0 +1,288 @@
+jobs:
+- job: Linux
+  pool:
+    vmImage: ubuntu-16.04
+  strategy:
+    matrix:
+      Plain:
+        CMakeArgs: '-DDEBUG=ON -DPROFILE=OFF -DBUILD_PYTHON_BINDINGS=OFF'
+        python.version: '2.7'
```

Comment:
Maybe we can avoid specifying `python.version` here since it isn't used for this particular build?

Reply:
Maybe, I guess we can skip the python dependent steps in the install dependencies task below if there is no python version specified.

Reply:
Yeah---I guess it doesn't hurt to include it anyway, I was just wondering. :+1: